tests/unit/src/test/scala/tests/markdown/ErrorSuite.scala

```diff
@@ -102,7 +104,21 @@ class ErrorSuite extends BaseMarkdownSuite {
   )
 
   checkError(
-    "already-defined",
+    "not-member-scala3".tag(OnlyScala3),
```

Comment:
Disclosure: before I realised there's a `compat = ...`, I was splitting out tests like this. I will remove most of those.

Some tests cannot be done this way, as the test's content itself is rejected by Scala 3. For those splitting is the only option, as far as I can see.